plugins/installer.plugin

```diff
@@ -80,20 +185,53 @@ $INSTALLED_PACKAGES
 EOF
 
   echo " * creating image.cpio.xz"
-  chroot $target sh -c "find / -xdev -depth -print | cpio --quiet -o -H newc -B" > $WORK_DIR/iso/image.cpio
-  $PXZ_CMD -T `nproc --ignore 1` -6 $WORK_DIR/iso/image.cpio 2>&1
+  cd $target
+  find . -xdev -depth -print | cpio --quiet -o -H newc -B > $WORK_DIR/iso/image.cpio
+  if [ $? -ne 0 ]; then
+    log_error "Error creating $WORK_DIR/iso/image.cpio"
+  fi
+  cd - >/dev/null
+  $PXZ_CMD -T `nproc --ignore 1` $WORK_DIR/iso/image.cpio 2>&1
```

Comment:
https://github.com/koalaman/shellcheck/wiki/SC2006

Reply:
agree, but this should go into a separate PR, changing this everywhere. for now it is consistent everywhere, and the issues mentioned there are not problematic here for some reason...